### ISSUE TITLE
Civil Penalty matrices

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#359]
We got this request via email, the documents had accessibility errors and need to be corrected.

**Comment by Maroyafaied** [comment]
Please post the attached civil penalty matrices to OE’s external webpage https://www.onrr.gov/compliance/civil-penalties.htm. Please note that Andrea has worked extensively to ensure that they are screen reader friendly and should meet the 508 compliance aspects as required. I have tested them and know they work well.

 

It is important that these reflect in a prominent position on our page. I was thinking it might be good to place them on the right hand column above “Penalties Collected by Fiscal Year”, listing them in a similar format:

 

Civil Penalty Assessment Tables

Curable Violations
Knowing or Willful Violations
 

Alternately, we could post them above the “References” heading near the center of the page. Please select what you believe will be easiest for users to find on the page. If you have any questions or feedback, please let me know.[
[Civil Penalty Assesment Table 2 (Knowing or Willful Violations).pdf](https://github.com/ONRR/onrr.gov-site/files/5554544/Civil.Penalty.Assesment.Table.2.Knowing.or.Willful.Violations.pdf)
[Civil Penalty Assesment Table 1 (Curable Violations).pdf](https://github.com/ONRR/onrr.gov-site/files/5554545/Civil.Penalty.Assesment.Table.1.Curable.Violations.pdf)
[Accessibility Report - Civil Penalty Assesment Table 1 (Curable Violations).pdf](https://github.com/ONRR/onrr.gov-site/files/5554549/Accessibility.Report.-.Civil.Penalty.Assesment.Table.1.Curable.Violations.pdf)
[Accessibility Report - Civil Penalty Assesment Table 2 (Knowing or Willful Violations).pdf](https://github.com/ONRR/onrr.gov-site/files/5554553/Accessibility.Report.-.Civil.Penalty.Assesment.Table.2.Knowing.or.Willful.Violations.pdf)



](url)

**Comment by Maroyafaied** [comment]
[Civil Penalty Assesment Table 1 (Curable Violations).pdf](https://github.com/ONRR/onrr.gov-site/files/5556030/Civil.Penalty.Assesment.Table.1.Curable.Violations.pdf)

**Comment by Maroyafaied** [comment]
[Uploading Civil Penalty Assesment Table 2 (Knowing or Willful Violations).pdf…]()

**Comment by Maroyafaied** [comment]
http://2017onrr/compliance/civil-penalties.htm

**Comment by Maroyafaied** [comment]
It's live now.
